src/main.cpp

```diff
@@ -768,6 +768,21 @@ void setup()
                 LOG_INFO("SX1280 Radio init succeeded, using SX1280 radio\n");
             }
         }
+    } else if (settingsMap[use_sx1268]) {
+        if (!rIf) {
+            LOG_DEBUG("Attempting to activate sx1268 radio on SPI port %s\n", settingsStrings[spidev].c_str());
+            LockingArduinoHal *RadioLibHAL = new LockingArduinoHal(SPI, spiSettings);
```

Comment:
I believe this now needs to use *LoraSPI to avoid potential conflicts with the display or touchscreen.

Reply:
Fixed.